index.html

```diff
@@ -4155,20 +4182,15 @@ <h2>Definition of Roles</h2>
 		<div class="role" id="legend">
 			<rdef>legend</rdef>
 			<div class="role-description">
-				<p>A visible name or caption for a group of related user interface components.</p>
-				<p>A element with role <code>legend</code> element can provide an accessible name for elements with grouping roles if it is progammatically associated with the element.  Authors MAY associate an element with role <code>legend</code> with an element by using one of two methods:</p>
+				<p>A visible name or caption for a group of related user interface components.  See synonym <rref>caption</rref>.</p>
+				<p>An element with role <code>legend</code> MAY provide an accessible name for <a href="#namefromcaptioncaption">elements that support the "caption/legend" naming technique</a> by using one of two methods:</p>
 				<ul>
-					<li>legend: The element contains an element with role <code>legend</code> element.  If the element contains more than one element with role <code>legend</code>, only the first descendant element with role <code>legend</code> is used for computing the accessible name.</li>
+					<li>legend: The element with role <code>legend</code> is the first child of the element it is naming.</li>
 					<li>reference: The element with role <code>legend</code> is referenced by the element it names via the <pref>aria-labelledby</pref> attribute.</li>
 				</ul>
-				<p>The legend method of naming is supported only if the element being named has one of the following grouping roles:</p>
-				<ul>
-					<li><pref>group</pref></li>
-					<li><pref>radiogroup</pref> </li>
-				</ul>
 				<p>Authors SHOULD ensure that:</p>
 				<ul>
-					<li>All elements with role <code>legend</code> are associated with one or more elements with grouping roles.</li>
+					<li>All elements with role <code>legend</code> are associated with one or more elements with roles that support the "caption/legend" naming technique.</li>
 					<li>When an element with role <code>legend</code> is activated by touch or a pointer, focus moves to the first focusable element in the associated group of user interface components.</li>
```

Comment:
as mentioned before, this functionality of moving focus should be removed as it's not how caption or legend work in HTML, and is not something authors SHOULD do.

Reply:
@scottaohara  I removed the authoring should's for the touch and pointer behaviors